head_first_design_patterns/pde-bakk/ch10_jsonparser/tests/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+set(EXEC test.out)
+project(JSON_tests CXX)
```

Comment:
instead of making multiple cmake projects you can just use `--target <target>` to avoid unnecessary compilations and have only one top-level cmake file to handle everything

Reply:
Done!